pkgs/development/libraries/appstream-glib/default.nix

```diff
@@ -1,19 +1,23 @@
-{ stdenv, fetchurl, pkgconfig, gettext, gtk3, intltool, glib
+{ stdenv, fetchFromGitHub, pkgconfig, gettext, gtk3, intltool, glib
 , gtk_doc, autoconf, automake, libtool, libarchive, libyaml
 , gobjectIntrospection, sqlite, libsoup, gcab, attr, acl, docbook_xsl
+, libuuid, json_glib
 }:
 
 stdenv.mkDerivation rec {
-  name = "appstream-glib-0.3.6";
+  name = "appstream-glib-0.5.11";
```

Comment:
I already had forgotten why I did that but it turns out that gnome-software 3.18 needs that version of appstream-glib.